Reed/Reader/views/ReaderView.swift

```diff
@@ -12,52 +12,44 @@ import SwiftUIPager
 
 struct ReaderView: View {
     @ObservedObject var viewModel: ReaderViewModel
-    @ObservedObject var definitionModalViewModel: DefinitionModalViewModel = DefinitionModalViewModel()
+    @ObservedObject var definitionViewModel: DefinitionViewModel = DefinitionViewModel()
     @State var page: Int = 0
     @State private var currentPosition: CGSize = .zero
     @State private var newPosition: CGSize = .zero
+    @State private var bottomSheetShown = false
```

Comment:
If the bottom sheet is always shown, do we need this value? Maybe we can just pass a constant.

Reply:
this turns true when the user drags the bottom sheet up

Reply:
In that case, maybe change the variable name to be something like `isBottomSheetExpanded` to avoid confusion in the future.